packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/react/integrationsteps.md

```diff
@@ -100,7 +104,11 @@ The user opens the email sent by Identity Engine. Create a dialog in your app wh
 
 <div class="half border">
 
-![Screenshot showing a page with an OTP input field.](/img/pwd-optional/pwd-optional-sign-up-js-react-email-verify.png)
+![A form with an OTP input field and a verify button.](/img/pwd-optional/pwd-optional-react-sign-up-verify-otp.png)
+
+<!--
+Source image: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Updated-Diagrams-for-Dev-Docs?node-id=2842%3A4925 pwd-optional-react-sign-up-verify-otp
```

Comment:
The name pwd-optional-react-sign-up-verify-otp has react inside of it.

Reply:
Again, yes it does. Was that not your intention?